src/pages/trade/api/book.ts

```diff
@@ -1,27 +1,28 @@
 import { useQuery } from '@tanstack/react-query';
 import { useRefetchOnNewBlock } from '@/shared/api/compact-block.ts';
-import { RouteBookResponse, RouteBookResponseJson } from '@/shared/api/server/book/types';
+import { RouteBookResponse } from '@/shared/api/server/book/types';
 import { deserializeRouteBookResponseJson } from '@/shared/api/server/book/serialization.ts';
+import { RouteBookApiResponse } from '@/shared/api/server/book';
+import { usePathSymbols } from '@/pages/trade/model/use-path.ts';
 
-export const useBook = (symbol1: string | undefined, symbol2: string | undefined) => {
+export const useBook = () => {
+  const { baseSymbol, quoteSymbol } = usePathSymbols();
```

Comment:
Realizing a few hooks are relying upon async data fetching for the path symbols. This is not necessary because they are synchronously available. So created a new `usePathSymbols();` for that.